content/Software Support/Installation/Error-bad-CPU-type-in-executable.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Error "bad CPU type in executable"'
+---
+
+Under certain conditions, startup or other operations of the Arduino development software might fail with a "`bad CPU type in executable`" error.
+
+This problem can be solved by one of the methods described below:
+
+## If your Mac computer has an Apple Silicon processor
+
+Some of the tools used by the Arduino development software are built for use on "Intel" processors. These can still be used on the [Apple Silicon](https://support.apple.com/en-us/HT211814) Mac computers thanks to a program named **Rosetta** that Apple provides.
+
+**Rosetta** is not pre-installed with macOS. If you have not already installed **Rosetta**, the tools built for use on "Intel" processors will fail to run with a "`bad CPU type in executable`" error.
+
+The solution is to install **Rosetta** by one of the following methods:
+
+### Installation Dialog
+
+When you start an application that requires it, macOS will show [a dialog offering installation of **Rosetta**](https://support.apple.com/en-us/HT211861). If you are presented with this dialog when starting the Arduino development software, click the ![Install button](img/symbol_install-rosetta.png) button and complete the installation.
+
+### Manual Installation
+
+If the tool built for use on "Intel" processors is run by the Arduino development software as a subprocess, macOS does not show a dialog offering installation of **Rosetta**. In this case you must manually install **Rosetta**:
```

Comment:
Avoiding the "Intel" term by instead taking about" requiring Rosetta" like under the previous heading.
 
```suggestion
If the tool requiring it is run by the Arduino development software as a subprocess, macOS does not show a dialog offering installation of **Rosetta**. In this case you must manually install **Rosetta**:
```